launchdarkly-server-sdk/src/evaluation.rs

```diff
@@ -118,7 +147,9 @@ impl FlagDetail {
                 continue;
             }
 
-            let detail = evaluate(store.to_store(), &flag, context, None);
+            let event_recorder = DirectPrerequisiteRecorder::new(key.clone());
```

Comment:
Could save a clone and take a reference, then give it back after we're done.. but seems not worth it.